### ISSUE TITLE
Integrate w/ Eventbrite API

### ISSUE THREAD

**Issue by kenhowardpdx** [csgpro/csgpro.com#156]
@JamieMikami what are your thoughts on using Eventbrite's API to list our events rather than the copy/paste approach we're doing now?

**Comment by kenhowardpdx** [comment]
Here's the API documentation: https://www.eventbrite.com/developer/v3/

**Comment by JamieMikami** [comment]
That would be nice.  Does eventbrite allow integration with Meetup.com and linkedIn?

**Comment by kenhowardpdx** [comment]
Maybe we should create a stand-alone service that aggregates events for display on the website. There isn't an integration between the services, so we could pull all the events from each service and normalize the data for use on the website or other services.

**Comment by JamieMikami** [comment]
yeah ... that just sounds like work.  Copy paste has been easy enough for me to avoid the work so far.  I agree we should make it better, just not sure how much time it is worth investing today.

Right now we need to post events on Meetup.com, then we take that and manually push to csgpro.com and then sometimes to LinkedIn and even less often to twitter.  I don't think we get that much visibility on anything other than Meetup.com today, so investing lots of your time is low priority to me.

**Comment by kenhowardpdx** [comment]
I like the idea of integrating w/ eventbrite mainly so the sales team can view events and possibly download (or view) the list of attendees. Let's revisit this in January when we might have an idea to how many events we'll have in the next year.
